csharp/test/Ice/optional/Test.ice

```diff
@@ -58,13 +58,12 @@ module ZeroC::Ice::Test::Optional
 
         void opSingleInInt(int? i1);
         void opSingleInString(string? i1);
-        int? opSingleOutInt();
-        string? opSingleOutString();
         int? opSingleReturnInt();
         string? opSingleReturnString();
 
         void opBasicIn(int i1, int? i2, string? i3, string i4);
-        (int? r1, int r2, int? r3, string? r4) opBasicInOut(int i1, int? i2, string? i3);
+
+        (int? r1, int r2, int? r3, string? r4) opBasicInReturnTuple(int i1, int? i2, string? i3);
```

Comment:
I guess you should rename more operations, since we no longer use the term 'in' for parameters. So for example:
```
void opInt(int? i1);
void opString(string? i1);
int? opReturnInt();
string? opReturnString();
void opBasic(...);
(...) opBasicReturnTuple(...);
```